variables.sh

```diff
@@ -6,17 +6,18 @@ NAMESPACE=${NAMESPACE:-namely}
 CONTAINER=${DOCKER_REPO}${NAMESPACE}
 LATEST=${1:false}
 
-GO_VERSION=${GO_VERSION:-1.14}
-GRPC_VERSION=${GRPC_VERSION:-1.39}
-GRPC_JAVA_VERSION=${GRPC_JAVA_VERSION:-1.39}
-GRPC_WEB_VERSION=${GRPC_WEB_VERSION:-1.2.1}
-GRPC_GATEWAY_VERSION=${GRPC_GATEWAY_VERSION:-2.0.1}
-UBER_PROTOTOOL_VERSION=${UBER_PROTOTOOL_VERSION:-1.3.0}
-SCALA_PB_VERSION=${SCALA_PB_VERSION:-0.11.0}
+GO_VERSION=${GO_VERSION:-1.17}
+GRPC_VERSION=${GRPC_VERSION:-1.42}
```

Comment:
since we're targeting 1.42, the release candidates should be named `v1.42_0-rc.X` instead of `v1.41_0-rc.X` moving forward (v1.41_0-rc.5 is misleading)

Reply:
(I've since Closed that `protorepo` PR, btw.)